samples/consul-ui/consul_nodes.json.erb

```diff
@@ -0,0 +1,63 @@
+<%
```

Comment:
Would it be possible to use a single file for services and nodes or is it too complicated?

Reply:
It's not too complicated but nodes file is bigger than services so I wanter to avoid loading twice as much data